test-suite/output-coqtop/DependentEvars2.out

```diff
@@ -0,0 +1,122 @@
+

```

Comment:
This file contains Windows line ends (CR + LF). Is that intended?

Reply:
I took them out, but it doesn't matter--the test runs fine either way.